docs/content/theory.rst

```diff
@@ -14,11 +13,119 @@ Here's how it works.
+All input arguments for the graph traversal, such as maximum calculation count or cut-off, can be passed to the ``TimexLCA`` instance.
+
+By default, only the foreground system is traversed, but nodes to be skipped during traversal can be added to a ``edge_filter_function``. 
+At each process, the graph traversal uses convolution to combine the temporal distributions of the consumed exchange and the process into the resoluting combined temporal distribution of the upstream producer of the exchange.
+
+Process timeline
+~~~~~~~~~~~~~~~~
+The graph traversal returns a timeline that lists the time of each technosphere exchange in the temporalized foreground system. 
+Exchanges that flow from same producer to the same consumer within a certain time-window (``temporal_grouping``) are grouped together. 
+This is done to avoid countless exchanges in the timeline, as the temporal distributions at exchange level can have arbitrary fine temporal resolutions while one may not have a similar temporal resolution at the database level. 
+We recommend aligning ``temporal_grouping``, which defaults to 'year', to align with the temporal resolution of the available databases.
+
+ For a simple example that consists of the following system: a process A that consumes an exchange b from a process B, which emits an emission x and both the exchange b and the emission x occur at a certain point in time 
+ 
+ (add flowchart dummy system)
+
+ the timeline would look like this:
+
+ 
+
+ +-------+-----------+----------+-----------------+
+ | time  | producer  | consumer | amount          |
+ +=======+===========+==========+=================+
+ | 0     | A         | n/a      | 1               |
+ +-------+-----------+----------+-----------------+
+ | 0     | B         | A        | 2 * 0.2 = 0.4   |
+ +-------+-----------+----------+-----------------+
+ | 1     | B         | A        | 2 * 0.8 = 1.6   |
+ +-------+-----------+----------+-----------------+
+
+Time mapping
+~~~~~~~~~~~~~~~~
+Based on the timing of the processes in the timeline, ``timex_lca`` matches the processes at the intersection between foreground and background to the best available background databases.
+Available matching strategies are: closest database or linear interpolation between two closest databases based on temporal proximity. The new best-fitting background producer(s) are mapped on the same name, reference product, location as the old background producer.
+
+Modified matrices
+~~~~~~~~~~~~~~~~~~~
+``timex_lca`` now modifies the technopshere and biosphere matrices using ``datapackages``.
+
+Technosphere matrix modifications:
 
-.. image:: data/matrix_dark.svg
+1. For each temporalized process in the timeline, a new process copy is created, which stores its timing in its ID and links to its new temporalized producers and consumers.
```

Comment:
"its id" is a bit misleading, no? Maybe mention the activity_time_mapping_dict here and that we keep track of the corresponding time of the process that way? or leave it out completely

Reply:
Rephrased now to "For each temporalized process in the timeline, a new process copy is created, which links to its new temporalized producers and consumers. The timing of the processes is stored in the ``activity_time_mapping_dict``, which maps the process ids to process timing."